common/config/elasticsearch.go

```diff
@@ -44,6 +46,10 @@ type (
 		Indices           map[string]string         `yaml:"indices"` //nolint:govet
 		LogLevel          string                    `yaml:"logLevel"`
 		AWSRequestSigning ESAWSRequestSigningConfig `yaml:"aws-request-signing"`
+		// Default is DefaultElasticsearchCloseIdleConnectionsInterval, set it to -1 to disable.
```

Comment:
why -1 to disable? Would it make sense to disable when it is 0 which is then by default and user don't have to do anything?